localenv/local-http-signatures/postman-scripts/preRequestSignaturesGrantRequest.js

```diff
@@ -37,20 +29,21 @@ pm.sendRequest(
   },
   (err, res) => {
     const keys = res.json()
-    pm.collectionVariables.set('keyId', keys.keys[0].kid)
+    pm.environment.set('keyId', keys.keys[0].kid)
 
     // Request Signature Headers
     pm.sendRequest(
       {
-        url: pm.collectionVariables.get('signatureUrl'),
+        url: pm.environment.get('signatureUrl'),
         method: 'POST',
         header: {
           'content-type': 'application/json'
         },
         body: {
           mode: 'raw',
           raw: JSON.stringify({
-            keyId: pm.collectionVariables.get('keyId'),
+            keyId: pm.environment.get('keyId'),
```

Comment:
should we specify that it's pfry's keyId? (even if they are the same for all the users)